shared/models/NaytonJarjestaja.ts

```diff
@@ -1,6 +1,15 @@
 import { types } from "mobx-state-tree"
+import { Organisaatio } from "./Organisaatio"
+import { EnrichOrganisaatioOid } from "./EnrichOrganisaatioOid"
 
-export const NaytonJarjestaja = types.model("NaytonJarjestaja", {
+const Model = types.model("NaytonJarjestaja", {
```

Comment:
Is this `"NaytonJarjestaja", ` required here?

Reply:
True, not needed. Forgot to remove it when I created composed model.

Reply:
Now that I checked, it seems that we have named compose model "root"-models with "Model" postfix. MST documentation says that models should have name for debugging purposes.